### ISSUE TITLE
QoS

### ISSUE THREAD

**Issue by derritter88** [jellyfin/jellyfin#2496]
Hi there,

I've moved from Plex to Jellyfin as I prefer the FOSS approach.
I just asked myself how a proper QoS could be configured?
So far I managed to create a policy according to the ports I have found here: [https://jellyfin.org/docs/general/networking/index.html](Jellyfin Networking).

Any proper solutions/ideas?

**Comment by JustAMan** [comment]
This should be discussed on main, server repo as it doesn't have anything to do with Web UI I think.

**Comment by hayer** [comment]
@JustAMan It doesn't have anything to do with the server either in my opinion. My vote is for this to be closed. The process is different between Sophos XG and pfSense, and opnSense, and so on. And this has to be setup on the network equipment, not in Jellyfin.

**Comment by JustAMan** [comment]
So maybe we should move it to https://github.com/jellyfin/jellyfin-docs/ and discuss as a documentation issue?

**Comment by derritter88** [comment]
@hayer : I agree if there are any updates in the documentation which ports to priorize this ticket can be closed.

**Comment by hayer** [comment]
Still doesn't feel like this belongs in any of the Jellyfin repositories at all. What do you want the documentation to say? HTTP data is sent in the ports you put in the configuration?

And setting ports for QoS fails if UPnP?

I'm still a bit confused about what the docs should say.
---
# Networking
## Quality of Service, QOS
The ports are the same listed in your configuration. If your running UPnP this will _probably_ not work. It all depends on your gear.

**Comment by derritter88** [comment]
I would more think like a non-IT-guy:
Networking
Quality of Service, QoS
For a better streaming "experience" (could be a different word) we recommend you to set up Quality of Service (QoS) on your router.
Please consult your router manual how to do that.

If you have not modified any ports please priorized the following one:

1234 TCP // HTTP
4321 TCP // HTTPS
1111 UDP // Broadcast for new devices
2222 UDP // Android TV stream
.... and so on.

**Comment by hayer** [comment]
Well that is my point "non-IT-guy" looking for information about QoS? Why should we educate him? My fear is that people think "QoS will solve my problem" then start asking questions about getting QoS set up.

How does QoS affect reverse proxies? Should that also be documented?

Port list can be found [here](https://github.com/jellyfin/jellyfin-docs/blob/master/general/networking/index.md) for the one that wants to write a PR.

**Comment by derritter88** [comment]
Why not?
From my point of view a good documenation is essential - also for non-IT-people.
More and more are moving to open source software but with limited knowledge how to do a proper setup.

I moved from Plex to Jellyfin last week but for Plex there are some useful tips on [(Reddit )](https://www.reddit.com/r/PleX/comments/4kq1t4/qos_for_plex/).
Would it make sense for you to just display a default QoS setup and ask the people to post their related QoS questions to Reddit or something similar?
